ml_service/pipelines/diabetes_regression_build_train_pipeline.py

```diff
@@ -31,14 +31,10 @@ def main():
         print("aml_compute:")
         print(aml_compute)
 
-    run_config = RunConfiguration(conda_dependencies=CondaDependencies.create(
-        conda_packages=['numpy', 'pandas',
-                        'scikit-learn', 'tensorflow', 'keras'],
-        pip_packages=['azure', 'azureml-core',
-                      'azure-storage',
-                      'azure-storage-blob',
-                      'azureml-dataprep'])
-    )
+    # Create a reusable run configuration environment
```

Comment:
1. The comment is little deceptive as it's not really a reusable environment
2. Can you please replicate similar changes to  diabetes_regression_build_train_pipeline_with_r.py
3. Please make use of AML environment to make is reusable

Reply:
Is it ok to keep 3 for a separate PR? We still need to agree on a strategy for avoiding interference between branches...

Reply:
it's ok

Reply:
1. done https://github.com/microsoft/MLOpsPython/pull/158/commits/921b65f9e1c7a19f8aa7271db7a2a9b910ebb7cf
2. done https://github.com/microsoft/MLOpsPython/pull/158/commits/921b65f9e1c7a19f8aa7271db7a2a9b910ebb7cf